guest-js/index.ts

```diff
@@ -89,12 +183,12 @@ export interface DownloadRecord {
 * Represents the state of a download operation.
 */
 export enum DownloadState {
-  UNKNOWN = 'UNKNOWN',
-  CREATED = 'CREATED',
-  IN_PROGRESS = 'IN_PROGRESS',
-  PAUSED = 'PAUSED',
-  CANCELLED = 'CANCELLED',
-  COMPLETED = 'COMPLETED'
+  UNKNOWN = 'unknown',
```

Comment:
I think IRL you mentioned why we needed to change these from the standard all-caps to camelCase. Would you be able to add a brief comment explaining why so that we 1) preserve that knowledge and 2) don't accidentally change it back later?

Reply:
Good idea. Done.